Simulator/Utils/RNG/Norm.h

```diff
@@ -21,6 +21,10 @@
 #ifndef _NORM_H_
 #define _NORM_H_
 
+#define  DEFAULT_M      0.0
+#define  DEFAULT_S      1.0
+#define  DEFAULT_SEED   0
```

Comment:
Use `const` declarations, rather than `#define`s, as the latter are untyped.

Reply:
Replaced to `static constexpr` declarations.